org.freedesktop.Sdk.Extension.php73.json

```
@@ -98,28 +96,54 @@
 				"--enable-xmlrpc",
 				"--enable-xmlwriter",
 				"--enable-xsl",
-				"--with-libzip",
 				"--enable-zlib",
-				"--with-zlib",
-				"--with-openssl"
+				"--with-argon2",
+				"--with-config-file-scan-dir=/usr/lib/sdk/php73/lib/ini:${HOME}/.config/php/7.3/ini",
```

Flatpak keeps config in `~/.var/appid/config`, `~/.config` is neither exposed nor persistent by default.
```suggestion
				"--with-config-file-scan-dir=/usr/lib/sdk/php73/lib/ini:${XDG_CONFIG_HOME}/php/7.3/ini",
```

---

I was a bit worried about doing that because that wouldn't allow the php configuration to be shared between different IDEs.

---

`~/.config` wouldn't be shared either, unless each IDE has access to host's `~/.config`. Also with per-IDE config, it could still be shared between IDEs by explicitly adding permission for it, via e.g. `--filesystem=xdg-config/php`.

---

I've updated it to use /var/config, good to you?

---

Probably we should also add a per-app, systemd-wide path here, like `/app/etc/php`, so bundled (copied from sdk extension to app) PHP would work, too.

---

Added that path.

---

You can achieve cross-flatpak config by manually adding override `--filesystem=xdg-config/php` for you IDEs.

---

Yeh. Is this all good now?

---

I believe so, but we can't say for sure until all covered cases are tested.

---

I'll mark this as resolve, if someone wants to set a shared custom location, they can just add PHP_INI_SCAN_DIR as an environment variable to the flatpak.